python/lsst/scarlet/lite/io/cube_component.py

```diff
@@ -19,68 +19,21 @@
 COMPONENT_TYPE = "cube"
 MigrationRegistry.set_current(COMPONENT_TYPE, CURRENT_SCHEMA)
 
+logger = logging.getLogger(__name__)
 
-class ComponentCube(Component):
-    """Dummy component for a component cube.
 
-    This is duck-typed to a `lsst.scarlet.lite.Component` in order to
-    generate a model from the component but it is currently not functional
-    in that it cannot be optimized, only persisted and loaded.
+class ComponentCube(CubeComponent):
```

Comment:
Missing the required [@deprecated](https://developer.lsst.io/stack/deprecating-interfaces.html) decorator, amongst other requirements for deprecation.